CHANGELOG.next.asciidoc

```diff
@@ -131,6 +131,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add support for Consul 1.9. {pull}24123[24123]
 - Add support for the MemoryPressure, DiskPressure, OutOfDisk and PIDPressure status conditions in state_node. {pull}23905[23905]
 - Store `cloudfoundry.container.cpu.pct` in decimal form and as `scaled_float`. {pull}24219[24219]
+- Add support for SASL/SCRAM authentication to the Kafka module. {pull}24810[24810]
```

Comment:
This line is in the "Breaking changes" section, could you please move it to the "Added" section? It'd be around line 938.